src/main/java/com/cedarsoftware/util/io/WriteOptions.java

```diff
@@ -721,46 +794,17 @@ private WriteOptions copyFieldsToAccessors(Class<?> clazz, Collection<String> fi
     }
 
     /**
-     * @param excludedFields Map of Class's mapped to Collection of String field names to exclude from written JSON.
+     * @param excludedFieldNames Map of Class's mapped to Collection of String field names to exclude from written JSON.
      * @return WriteOptions for chained access.
      */
-    public WriteOptions addExcludedFields(Map<Class<?>, Collection<String>> excludedFields) {
+    public WriteOptions addExcludedFields(Map<Class<?>, Collection<String>> excludedFieldNames) {
```

Comment:
For every call to addExcludedFields() below, it is going to perform a this.clearCaches().  If any of the prior calls to addExcludedFields(Class, Collection) add content to caches, it will get cleared for each iteration of the loop.  I don't think it does any work that is cached that would be cleared, but it is something to watch.  Also, you could remove the "throwIfBuilt()" call at the top, knowing the "addExcludedFields()" also checks that.  But, I don't mind it staying, from a pattern/consistency pov.